packages/neo4j-driver/examples/node.js

```diff
@@ -20,9 +20,9 @@
 var neo4j = require('neo4j')
 
 var query = [
-  'MERGE (alice:Person {name:{name_a},age:{age_a}})',
-  'MERGE (bob:Person {name:{name_b},age:{age_b}})',
-  'CREATE UNIQUE (alice)-[alice_knows_bob:KNOWS]->(bob)',
+  'MATCH (alice:Person {name:{name_a},age:{age_a}})',
+  'MATCH (bob:Person {name:{name_b},age:{age_b}})',
+  'MERGE (alice)-[alice_knows_bob:KNOWS]->(bob)',
```

Comment:
```suggestion
  'MERGE (alice:Person {name:{name_a},age:{age_a}})',
  'MERGE (bob:Person {name:{name_b},age:{age_b}})',
  'MERGE (alice)-[alice_knows_bob:KNOWS]->(bob)',
```


`alice` and `bob` might don't exist, so we should use `MERGE` instead of `MATCH` in this case.

Reply:
Same comment for `age` and `ON CREATE`